ftw/testing/tests/test_freezer.py

```diff
@@ -212,6 +212,6 @@ def test_commit_works_with_transactions(self):
 
         thedate = self.layer['portal'].current_date
         self.assertEquals(datetime.datetime(2015, 7, 22, 11, 45, 58), thedate)
-        self.assertEquals('datetime.datetime',
+        self.assertEquals('ftw.testing.freezer.FrozenDateTime',
```

Comment:
Why did that change?
Is it a problem that it did change?

I remember that we had some problems at one point because e.g. `type(value) == datetime.datetime` was no longer `True`.

Reply:
Discussion: we are going to ship that as it is and fix it when we run into errors.